src/core/streams/stream_actions.js

```diff
@@ -40,6 +40,10 @@ export const StreamActions = {
   },
 
   morph() {
-    morph(this)
+    const morph = this.hasAttribute("children-only") ?
+      morphChildren :
+      morphElements
+
+    this.targetElements.forEach((targetElement) => morph(targetElement, this.templateContent))
```

Comment:
This change adheres to the current style.

As a related change as a follow-up, https://github.com/hotwired/turbo/pull/1240 proposes extending the existing `[action="replace"]` and `[action="update"]` with new `[method="morph"]` attributes named in a way that corresponds to the value used in the [`<meta name="turbo-refresh-method" content="morph">`](https://turbo.hotwired.dev/handbook/page_refreshes#morphing) element.